proof/refine/ARM/Invariants_H.thy

```diff
@@ -3868,12 +3890,16 @@ private lemma ko_at_defn_ko_wp_at':
   by (clarsimp simp: ko_at'_defn_def obj_at'_real_def
                      ko_wp_at'_def project_inject)
 
-method normalise_obj_at' =
+(* VER-1364: normalise_obj_at' sometimes doesn't normalise ob_at' unless you use it twice. *)
```

Comment:
If this is a complete issue, we usually don’t leave such comments in the theory file. You can record that it’s completed in the JIRA ticket itself, and you can certainly comment on the PR instead.

Reply:
Do you mean "completed" issue? This is a hacky temporary fix - the real fix should be to figure out why `normalise_obj_at'_step` doesn't perform all the required work in one go, hence the outstanding comment.